DESIGN.md

```diff
@@ -216,6 +228,10 @@ Many features can be tested adequately by only looking at the list of mutants pr
 
 Although we primarily want to test the public interface (which is the command line), unit tests can be added in a `mod test {}` within the source tree for any behavior that is inconvenient to exercise from the command line.
 
+### Nextest tests
+
+cargo-mutants tests require `nextest` to be installed.
```

Comment:
May also want a link here. (And I think based on the PR you don't need to run cargo-mutants' tests with `cargo nextest run`, but nextest is invoked by one of the tests. Might be useful to make that clearer.)

Reply:
Done, but I moved this to CONTRIBUTING.md